src/operations/executors/convolution_executor.ts

```diff
@@ -90,6 +90,24 @@ export let executeOp: OpExecutor =
               [stride[1], stride[2]], pad as 'valid' | 'same',
               dataFormat as 'NHWC' | 'NCHW', [dilations[0], dilations[1]])];
         }
+        case 'conv3d': {
+          const stride =
+              getParamValue('strides', node, tensorMap, context) as number[];
+          const pad = getParamValue('pad', node, tensorMap, context);
+          const dataFormat =
+              (getParamValue('dataFormat', node, tensorMap, context) as string)
+                  .toUpperCase();
+          const dilations =
+              getParamValue('dilations', node, tensorMap, context) as number[];
+          return [tfc.conv3d(
+              getParamValue('x', node, tensorMap, context) as tfc.Tensor4D |
+                  tfc.Tensor<tfc.Rank.R5>,  // TODO: change type to tfc.Tensor5D
+              getParamValue('filter', node, tensorMap, context) as
+                  tfc.Tensor<tfc.Rank.R5>,  // TODO: change type to tfc.Tensor5D
+              [stride[1], stride[2], stride[3]], pad as 'valid' | 'same',
```

Comment:
TF op has strides and dilation array in length of 5, tfjs-core only support length of 3, seems we only support dimension of [depth, height, width].
Should both of strides and dilation values selected from indices of 1, 2, 3?

Reply:
yes, i have changed the strides and dilations to both use indices 1, 2, 3. fyi according to the [`tf.nn.conv3d` docs](https://www.tensorflow.org/api_docs/python/tf/nn/conv3d), indices 0 and 4 in strides and dilations must be 1.